lib/stdlib/src/Makefile

```diff
@@ -130,7 +130,7 @@ HRL_FILES= \
 	../include/qlc.hrl \
 	../include/zip.hrl
 
-INTERNAL_HRL_FILES= dets.hrl
+INTERNAL_HRL_FILES= dets.hrl erl_tar.hrl
 
```

Comment:
You should add a dependency from `erl_tar.beam` to `erl_tar.hrl` at the bottom of this makefile.